src/helpers/JSON-performance-report-parser.helper.js

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs';
+
+const getReport = (fileName, path) => {
+  return JSON.parse(fs.readFileSync(`${path}/${fileName}`, 'utf8'));
+};
+
+class JSONPerformanceReportParser {
+  constructor(path) {
```

Comment:
Hm... I'm thinking if splitting parameters to `path` and `fileName` is necessary. Does it have any purpose?
I guess it would be easier to have just full path to the file passed to the constructor

Reply:
Actually yes, because `fileName` is a global variable given in performance step and I wanted to keep this `path` default. 
So currently I can easily mock `path` and `fileName` in unit tests.

Reply:
I would rather keep it separated. It would much easier to move it to kakunin.config.js at some point.

Reply:
Oh, good point with moving to config file.

Reply:
I kept it separated and moved `default path` to the constructor of JSONParser class.